src/System.Runtime.InteropServices.RuntimeInformation/src/Resources/Strings.resx

```diff
@@ -61,4 +61,7 @@
   <data name="Argument_EmptyValue" xml:space="preserve">
     <value>Value cannot be empty.</value>
   </data>
+  <data name="PlatformNotSupported_RuntimeInformation" xml:space="preserve">
+    <value>RuntimeInformation is a stub implementation assembly for PCLs, where runtime rids are not supported by packages.config.</value>
```

Comment:
To understand this error message, you need to know what both "PCLs" and "runtime rids" are.